src/utils/arn.ts

```diff
@@ -1,20 +1,57 @@
+import { stringify } from "querystring";
+
 /** Parse properties of the ARN into an object */
-export function parseLambdaARN(functionARN: string) {
-  // Disabling variable name because account_id is the key we need to use for the tag
+
+interface Tags {
+  // Disabling variable name because accountId is the key we need to use for the tag
   // tslint:disable-next-line: variable-name
-  const [, , , region, account_id, , functionname] = functionARN.split(":", 7);
-  return { region, account_id, functionname };
+  account_id: string;
+  region: string;
+  functionname: string;
+  executedversion?: string;
+  resource?: string;
+}
+
+export function parseLambdaARN(arn: string, version?: string) {
+  let region: string | null = null;
+  // tslint:disable-next-line: variable-name
+  let account_id: string | null = null;
+  let functionname: string | null = null;
+  let alias: string | null = null;
+
+  const splitArn = arn.split(":");
+  // If we have a version or alias let's declare it
+  splitArn.length === 8
+    ? ([, , , region, account_id, , functionname, alias] = splitArn)
```

Comment:
With node you can just write `[, , , region, account_id, , functionname, alias] = splitArn;`, and alias will be null if there aren't enough values to unpack.